packages/upswyng-server/src/components/ScheduleSelector.svelte

```diff
@@ -259,132 +258,122 @@
+        <div class="column is-one-fifth">
           <label class="label" for="from-time">From</label>
           <Select
             bind:selectedValue={weeklyRepeatState.fromTime}
             items={Time.options.filter(o => !o.isNextDay)} />
         </div>
-        <div class="column">
+        <div class="column is-one-fifth">
           <label class="label" for="to-time">To</label>
           <Select
             bind:selectedValue={weeklyRepeatState.toTime}
             items={Time.options} />
         </div>
       </div>
-      <div class="content">
-        {#if isExpanded}
-          <header class="more-less-selector">
-            <!-- svelte-ignore a11y-missing-attribute -->
-            <a
-              on:click|preventDefault={() => toggleExpanded()}
-              class="has-text-grey is-uppercase is-size-6 has-text-weight-bold">
-              <span>Fewer Options</span>
-              <!-- svelte-ignore a11y-missing-attribute -->
-              <span class="icon is-small">
-                <i class="fas fa-chevron-up" aria-hidden="true" />
-              </span>
-            </a>
-          </header>
-        {:else}
-          <header class="more-less-selector">
-            <!-- svelte-ignore a11y-missing-attribute -->
-            <a
-              on:click|preventDefault={() => toggleExpanded()}
-              class="has-text-grey is-uppercase is-size-6 has-text-weight-bold">
-              <span>More Options</span>
-              <!-- svelte-ignore a11y-missing-attribute -->
-              <span class="icon is-small">
-                <i class="fas fa-chevron-down" aria-hidden="true" />
-              </span>
-            </a>
-          </header>
-        {/if}
+
+      <div class="control field">
+        <span class="label" id="day-repeat">These days repeat</span>
+        <label class="radio" aria-describedby="day-repeat">
+          <input
+            type="radio"
+            name="repeats"
+            checked={certainWeekRepeatState.everyWeek}
+            on:click={() => {
+              certainWeekRepeatState = { ...certainWeekRepeatStateDefault };
+            }} />
+          weekly
+        </label>
+        <label class="radio" aria-describedby="day-repeat">
+          <input
+            type="radio"
+            name="repeats"
+            checked={!certainWeekRepeatState.everyWeek}
+            on:click={() => {
+              certainWeekRepeatState = { ...certainWeekRepeatState, everyWeek: false, [1]: true };
+            }} />
+          monthly
+        </label>
       </div>
-      {#if isExpanded}
-        <div class="content">
-          <div class="field">
-            <label class="label">
-              Certain occurances of the day in a month
-            </label>
-          </div>
-          <div class="field has-addons">
+      {#if !certainWeekRepeatState.everyWeek}
+        <div class="field">
+          <label class="label">How often do these days repeat?</label>
```

Comment:
This label doesn't make sense.

Q: "How often do these days repeat?"
A: "Third"

🤯

Reply:
Ya that's fair. What about this [update](https://github.com/CodeForBoulder/upswyng/pull/333/commits/d3ec52e76cec9d76606149c167a7f3e16e09bac9)?
<img width="772" alt="Screen Shot 2020-05-06 at 11 26 12 AM" src="https://user-images.githubusercontent.com/6598084/81208551-7cc99300-8f8c-11ea-89c4-e9565426276f.png">

Reply:
That looks pretty good. I just remembered I pulled some screenshots from google calendar I'll post them up. This seems better though